packages/manager/modules/pci/src/projects/project/storages/containers/container/addUser/addUser.html

```diff
@@ -0,0 +1,41 @@
+<oui-modal
+    data-heading="{{ ('pci_projects_project_storages_containers_container_addUser_'+ ($ctrl.archive ? 'archive' : 'object') + '_title') | translate }}"
+    data-primary-action="$ctrl.addUserStorage()"
+    data-primary-label="{{ :: 'pci_projects_project_storages_containers_container_addUser_submit_label' | translate }}"
+    data-primary-disabled="$ctrl.isLoading"
+    data-secondary-action="$ctrl.goBack()"
+    data-secondary-label="{{:: 'pci_projects_project_storages_containers_container_addUser_cancel_label' | translate }}"
+    data-on-dismiss="$ctrl.goBack()"
+    data-loading="$ctrl.isLoading"
+>
+    <div data-ng-if="$ctrl.container">
+        <div data-ng-if="$ctrl.addUserStep === 0">
+            <oui-radio-group
+                model="$ctrl.selectedUser"
+                data-on-change="$ctrl.onUserSelect()"
+            >
+                <oui-radio
+                    ng-repeat="user in $ctrl.availableUsers track by $index"
+                    value="user.id"
+                    ><span data-ng-bind="user.description"></span
+                ></oui-radio>
+            </oui-radio-group>
+        </div>
```

Comment:
I think we will have to provide a way to select a different user too... For example, I may have selected the wrong user in step 1 and I might want to go back to step 1 to reselect the user. I think we will need a back button in step 2. What do you think?

Reply:
To achieve this, we have to implement a custom modal with `"modal-footer"`. Since design does not specify a back button, i think this should be okay. User can cancel and restart the step 1 (only 2 steps, so should be okay).

Reply:
I agree with Ganesh comment. All steppers/wizards must support to go back to previous step. oui-stepper has it inbuilt. Can you please add a back button to go back to step 1?

Reply:
Have added a back button now to go back to first step.